generate_sample_file.py

```diff
@@ -141,38 +144,24 @@ def generate_sample_file(self, output_file_path: Path, treatment_code_quantities
             for treatment_code in treatment_code_quantities:
                 for _ in range(treatment_code["quantity"]):
                     if treatment_code["treatment_code"][:3] == "SPG":
-                        self._write_estab_case(writer, sequential_uprn, treatment_code, address_type='SPG')
+                        self._write_spg_or_ce_case(writer, sequential_uprn, treatment_code, address_type='SPG',
+                                                   address_level=treatment_code["address_level"])
                         continue
 
-                    # Randomly decide whether to create a CE case or not
-                    if self.random_1_in_11():
-                        self._write_estab_case(writer, sequential_uprn, treatment_code, address_type='CE')
+                    if treatment_code["treatment_code"][:2] == "CE":
```

Comment:
Any idea if we're gonna get CE treatment codes soon so we can put it in the sample generator? At the moment it'll just generate HH and SPG

Reply:
no idea

Reply:
no idea